tests/test_common.c

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file test_common.h
+ * @author Irfan
+ * @brief common header file for all tests to facilitate uniform logging format
+ *
+ * @copyright
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+#define _GNU_SOURCE
+
+#include <pthread.h>
+#include <assert.h>
+#include <stdio.h>
+#include <time.h>
+#include "sysrepo.h"
+#include "test_common.h"
+
+static void _test_log_msg(sr_log_level_t level, const char *message, const char* prefix)
+{
+    const char *severity;
+
+    switch (level) {
+    case SR_LL_ERR:
+        severity = "ERR";
+        break;
+    case SR_LL_WRN:
+        severity = "WRN";
+        break;
+    case SR_LL_INF:
+        severity = "INF";
+        break;
+    case SR_LL_DBG:
+        /*severity = "DBG";
+        break;*/
+        return;
+    case SR_LL_NONE:
+        assert(0);
+        return;
+    }
+    struct timespec ts;
+    clock_gettime(CLOCK_REALTIME, &ts);
+    ts.tv_sec %= 1000;
+    ts.tv_nsec /= 1000;
+    fprintf(stderr, "%03ld.%06ld [%ld][%lu][%s] %s: %s\n", ts.tv_sec, ts.tv_nsec,
+                    (long)getpid(), (unsigned long)pthread_self(), severity,
```

Comment:
Coding style, this should 2x indent, indent is 4x space so it should be 8x space, this is 16x space.